### ISSUE TITLE
a few non-words

### ISSUE THREAD

**Issue by helmadik** [dspinellis/word-master-ancient-greek#1]
like ΑΒΓΔΕ, ΒΕΡΓΚ. (Euclid, faulty encoding of textual notes at Perseus)

**Comment by helmadik** [comment]
I'll see if I can flatten my wordlist with your scripts, and filter out the Euclid and the weird Old Testament names.

**Comment by dspinellis** [comment]
Oh, I see, geometric point sequences by Euclid. What a fascinating journey to end in a Wordly list!

**Comment by helmadik** [comment]
yes, and 5-letter philologists (Bergk) who were badly encoded in the texts. But also weird Old Testament names that don't follow normal Greek phonology. I'll try to come up with an alternative list.

**Comment by dspinellis** [comment]
Thank you!  given that this lacks frequency information I'll need to join this with the frequencies from the Perseus Project to derive the 1800 words used for setting questions.  The new list contains 10863 words whereas the original about 13800 words.  Do you think the new so much better than the original one that it should replace it?  Alternatively we could use the new list to sanity-check the original one in order to remove non-Greek words.  It would require manual reviewing of about 3000 words.

**Comment by helmadik** [comment]
My sense is that in the dirty xml there were a lot of words with apostrophes, also hyphenated words across lines. I'd suggest at least doing the sanity check on the *answers* file - don't much care if people try non-words in the responses (people have to type them in to see them); but given that some critical apparatus notes may proliferate and make it into the answers list, I would prioritize cleaning that one? If you can do the filtering, I'm happy to eyeball that shorter list. Although even 3000 is not a lot. It's just that it's a little harder to spot in all-caps -just more unfamiliar. I can do some batches and nominate candidates for removal:-)

**Comment by dspinellis** [comment]
If you list has no false positives, then I can use it to clean up the answers list, together with some manual cleaning.

**Comment by helmadik** [comment]
the only thing I can immediately think of, if you find words in my list not
in the Perseus list, is that I have some Anna Komnena in there. But those
words are all in my database as tokens with a parse (e.g. masc acc sg noun)
and a dictionary entry. Should be safe.

In the full list, you can delete the words that end in T for instance. Or M
(a fair amount of Latin has come in). Etc.

Helma Dik
Department of Classics
University of Chicago
